frontend/src/components/base/Logo.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <v-toolbar-items>
+    <v-btn text class="px-2"
+           min-width="0" rounded
+           exact :to="{ name: 'home'}">
+      <v-toolbar-title>
+        <i>🏕</i>️<span v-if="text" class="ml-4 mr-2">eCamp</span>
+      </v-toolbar-title>
+    </v-btn>
+    <slot/>
```

Comment:
```suggestion
    <slot />
```
Könntest du noch `npm run lint --fix` bzw. `docker-compose exec frontend npm run lint --fix` drüber laufen lassen? Hat einige Linter-Fehler bei mir.

Reply:
Habe ich gemacht. Ist aber beides akzeptiert.

Reply:
Hm wie kann das sein? Dann stimmt etwas mit unserer (meiner?) ESLint Konfiguration nicht